contents/articles/roll-call-brett/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Roll call: Brett Langdon"
+author: brett-langdon
+date: 2015-06-21
+template: article.jade
+---
+
+Continuation of the series to look behind the curtain into the engineering team at [Underdog.io][]
+
+[Underdog.io]: https://underdog.io
+
+---
+
+<img alt="Photo of Brett" src="brett.jpg" style="width: auto; float: left; padding-right: 10px;" />
+
+#### Who are you and what do you do at Underdog.io?
+My name is Brett Langdon and I am senior software engineer at [Underdog.io][].
+
+Since we are still a small team, we all get to wear many hats and get our hands dirty with every aspect of our platform. Some days I am either writing backend Python or Node.js, writing frontend HTML/CSS/JS or staring at either our [AWS][] management console or [Datadog][] dashboards.
```

Comment:
One more suggestion:

- You are using "or" as an "or" but also slashes. It might help to be more consistent with that =/